### PR TITLE
[Website] hero section

### PR DESCRIPTION
**Pull request by Codebmk**
#### Summary of Changes (What does this PR do?)
- Implemented new hero section

#### Is this change ready to hit production in its current state?
- [x] Yes
- [ ] No

#### Status of maturity (all need to be checked before merging):

- [x] I consider this code done
- [ ] I've tested this locally
- [ ] This branch has been LGTM'ed by a reviewer

#### How should this be manually tested?
* Please include the steps to be done inorder to setup and test this PR.

#### What are the relevant tickets?
- [WEB-215](https://airqoteam.atlassian.net/browse/WEB-215)

#### Screenshots
hero-sm
hero

### CODE REVIEW

File: website/frontend/src/components/Hero.js [hunk 1]
```diff
@@ -47,9 +48,7 @@ const Hero = () => {
             <div className="hero-content">
                 <div>
                 <p className="hero-title">Clean air for <br />all African cities </p>
-                <p className="hero-sub">We empower communities with accurate, hyperlocal and timely air quality data to
-                    drive air pollution mitigation actions.
-                </p>
+                <p className="hero-sub"> <span style={{color:"#135DFF"}}>“9 out of 10 people breathe polluted air”.</span> <br/>We empower communities with accurate, hyperlocal and timely air quality data to drive air pollution mitigation actions.</p>
                 <div className="hero-buttons">
                     <Button label="Explore data" />
                     <Button className="button-get-involved" label="Get Involved" />
```
Comment (FaithDaka) [inline website/frontend/src/components/Hero.js:54]: The buttons should be aligned to the center in the mobile design. They seem to be left-aligned.

**Review by FaithDaka — CHANGES_REQUESTED** [review]

File: website/frontend/src/components/Hero.js (hunk 1 above)
Comment (Codebmk) [reply website/frontend/src/components/Hero.js:54]: > The buttons should be aligned to the center in the mobile design. They seem to be left-aligned. Unless that's an issue with browser responsiveness?

@FaithDaka I have fixed that. Please re-review

**Review by FaithDaka — APPROVED** [review]

**Review by 123MwanjeMike — APPROVED** [review]
Thanks @Codebmk 
LGTM

**Review by ja-odur — APPROVED** [review]
LGTM

**Pull request merged**